aspnetcore/signalr/background-service/sample/Clients.ConsoleTwo/ClockHubClient.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using HubServiceInterfaces;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Clients.ConsoleTwo
+{
+    public partial class ClockHubClient : IClock, IHostedService
+    {
+#region ClockHubClientCtor
+        private readonly ILogger<ClockHubClient> _logger;
+        private HubConnection _connection;
+
+        public ClockHubClient(ILogger<ClockHubClient> logger)
+        {
+            _logger = logger;
+            
+            _connection = new HubConnectionBuilder()
+                .WithUrl(Strings.HubUrl)
+                .Build();
+
+            _connection.On<DateTime>(Strings.Events.TimeSent, 
+                dateTime => _ = ShowTime(dateTime));
+        }
+
+        public Task ShowTime(DateTime currentTime)
+        {
+            _logger.LogInformation($"{currentTime.ToShortTimeString()}");
+
+            return Task.CompletedTask;
+        }
+#endregion
+
+#region StartAsync
+        public async Task StartAsync(CancellationToken cancellationToken)
+        {
+            while (true)
```

Comment:
Maybe add a comment here describing the loop here is just to make sure the server is running.